sklearn/tree/export.py

```diff
@@ -120,7 +120,8 @@ def export_graphviz(decision_tree, out_file="tree.dot", max_depth=None,
 
     proportion : bool, optional (default=False)
         When set to ``True``, change the display of 'values' and/or 'samples'
-        to be proportions and percentages respectively.
+        to be proportions and percentages respectively. When set to 'both',
```

Comment:
The only other options would be to either add a "values" parameter that can be true or false to control them independently, or to deprecate "proportions" and add a new parameter that can be "proportions", "counts" or "both".
I think your option is the best.